CHANGELOG.md

```diff
@@ -81,7 +89,8 @@
 - Fix race condition when installing sandbox and running multiples tests at the same time. https://github.com/near/workspaces-rs/pull/46
 
 
-[Unreleased]: https://github.com/near/workspaces-rs/compare/0.4.0...HEAD
+[Unreleased]: https://github.com/near/workspaces-rs/compare/0.4.1...HEAD
+[0.4.1]: https://github.com/near/workspaces-rs/compare/0.4.0-no-eh...0.4.1
```

Comment:
```suggestion
[0.4.1]: https://github.com/near/workspaces-rs/compare/0.4.0...0.4.1
```

typo?

Reply:
this was to exclude the error handling PR. Not sure how to express that in the comparison besides starting from a commit hash that doesn't have it

Reply:
why would this need a different starting tag? doesn't 0.4.1 continue from 0.4.0? Or was 0.4 done manually to exclude a commit?

Reply:
yeah this is kinda funky. It just so happens the error handling commit is right after 0.4.0 commit, which means we can skip it and start from the commit after.

The actual crate release will just be cherry picked commits though, so the CHANGELOG is more of me trying to not include it within the reference frame of 0.4.0...0.4.1. The above is what I came up with by having a special tag at the start of after error handling just so we don't actually show it in the comparison link, but let me know if there's a better way of doing this

Reply:
Either way, if you've cherry-picked or removed that commit, wouldn't the history include 0.4.0 tag? I guess I'm just more curious about how this release will be created, since releasing this commit merged would include error handling?

Reply:
it's not going to be from the main branch but on a separate branch with all the cherry picked commits. Probably could just reference that branch instead for the comparison

Reply:
but what I'm saying is that you could compare the two tags, since the branch would have 0.4.0 in the commits? This isn't a blocking comment btw, feel free to do the release and merge if you're sure

Reply:
I think I mostly get what you're saying, so just to be clear you're saying we have something like:
`https://github.com/near/workspaces-rs/compare/0.4.0...0.4.1` where 0.4.1 is the tagged via the separate branch (with all the cherry-picked commits) right?

Reply:
Is that not what you were planning on doing anyway? I'm confused. Just do what you were planning on doing, was just trying to help avoid an unnecessary new tag